### PR TITLE
(SIMP-3282) Fix CentOS repo edge cases & comments

### PR DESCRIPTION
**Pull request by op-ct**
This patch fixes some edge cases that missed disabling CentOS repos, and syncs the structure of the CentOS 6 & 7 kickstart template files with each other.  In both files, the prelink logic has been moved
into the FIPS comment block.

SIMP-3282 #close

### CODE REVIEW

File: build/distributions/CentOS/6/x86_64/DVD/ks/pupclient_x86_64.cfg [hunk 1]
```diff
@@ -188,18 +189,17 @@ fi
 %post
 ostype="#LINUXDIST#"
 if [ $ostype == "CentOS" ]; then
-  sed -i '/enabled=/d' /etc/yum.repos.d/CentOS-Base.repo;
-  sed -i '/\[.*\]/ a\
-enabled=0' /etc/yum.repos.d/CentOS-Base.repo;
+  sed -i -e '/enabled=/d' -e '/^\[.*\]/ aenabled=0' /etc/yum.repos.d/CentOS-Base.repo;
```
Comment (jeefberkey) [inline build/distributions/CentOS/6/x86_64/DVD/ks/pupclient_x86_64.cfg]: `aenabled`

File: build/distributions/CentOS/7/x86_64/DVD/ks/pupclient_x86_64.cfg [hunk 2]
```diff
@@ -189,18 +189,17 @@ fi
 %post
 ostype="#LINUXDIST#"
 if [ $ostype == "CentOS" ]; then
-  sed -i '/enabled=/d' /etc/yum.repos.d/CentOS-Base.repo;
-  sed -i '/\[.*\]/ a\
-enabled=0' /etc/yum.repos.d/CentOS-Base.repo;
+  sed -i -e '/enabled=/d' -e '/^\[.*\]/ aenabled=0' /etc/yum.repos.d/CentOS-Base.repo;
```
Comment (jeefberkey) [inline build/distributions/CentOS/7/x86_64/DVD/ks/pupclient_x86_64.cfg]: `aenabled`

File: build/distributions/CentOS/6/x86_64/DVD/ks/pupclient_x86_64.cfg (hunk 1 above)
Comment (jeefberkey) [reply build/distributions/CentOS/6/x86_64/DVD/ks/pupclient_x86_64.cfg]: This was sed magic and it should be fine, but maybe a comment explaining why `aenabled=0` isn't a typo would be great

File: build/distributions/CentOS/6/x86_64/DVD/ks/pupclient_x86_64.cfg [hunk 3]
```diff
@@ -219,6 +219,19 @@ fi
...
+If this host is not autosigned by Puppet, sign your Puppet certs to begin bootstrap.
+Otherwise, it should already be running! Tail /root/puppet.bootstrap.log for details.
+Wait for completion and reboot.
+
+To remove this message, delete /root/.bootstrap_msg" > /root/.bootstrap_msg
+sed -i "2i if [ -f /root/.bootstrap_msg ]\nthen\n  cat /root/.bootstrap_msg\nfi" /root/.bashrc
+source /root/.bashrc
+
+# Enable wait-online
+systemctl enable NetworkManager-wait-online
```
Comment (trevor-vaughan) [inline build/distributions/CentOS/6/x86_64/DVD/ks/pupclient_x86_64.cfg]: This won't work in CentOS 6

File: build/distributions/CentOS/7/x86_64/DVD/ks/pupclient_x86_64.cfg (hunk 2 above)
Comment (op-ct) [reply build/distributions/CentOS/7/x86_64/DVD/ks/pupclient_x86_64.cfg]: Curiously, this actually works.

File: build/distributions/CentOS/6/x86_64/DVD/ks/pupclient_x86_64.cfg (hunk 1 above)
Comment (op-ct) [reply build/distributions/CentOS/6/x86_64/DVD/ks/pupclient_x86_64.cfg]: It _also_ works with `-e '/^\[.*\]/a enabled=0'`, so I've changed it to that. 👍

File: build/distributions/CentOS/7/x86_64/DVD/ks/pupclient_x86_64.cfg [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
-# #ROOTPASS# - Your hashed root password
+# #BOOTPASS# - Your SHA-512 hashed bootloader password
+# #ROOTPASS# - Your SHA-512 hashed root password
 # #KSSERVER# - The IP address of your Kickstart server
 # #YUMSERVER# - The IP address of your YUM server
 # #LINUXDIST# - The LINUX Distribution you are kickstarting
 #        - Current CASE SENSITIVE options: RedHat CentOS
+#
+# To generate the BOOTLOADER hash:
+#  > grub-crypt
```
Comment (jeannegreulich) [inline build/distributions/CentOS/7/x86_64/DVD/ks/pupclient_x86_64.cfg]: This should be grub2-mkpasswd-pbkdf2

**Pull request merged**